src/client/article.ts

```diff
@@ -53,14 +53,20 @@ function insertAds(): void {
 
         const observer = new MutationObserver(callback);
         observer.observe(targetNode, config);
+
+        try {
+            (document as any).fonts.ready.then(() => nativeClient.updateAdverts(getAdSlots()))
```

Comment:
classed as an experimental API so it's not in Typescript definitions.
Seems to work on iOS and says it's supported on Android though https://developer.mozilla.org/en-US/docs/Web/API/CSS_Font_Loading_API

Reply:
I'm up for using this and seeing how it works

Reply:
We could introduce a subset of the types [suggested here](https://github.com/Microsoft/TypeScript/issues/30984#issuecomment-581953895) to avoid having to type this as `any`.

```ts
interface FontFaceSet {
    readonly ready: Promise<FontFaceSet>;
}

declare global {
    interface Document {
        fonts: FontFaceSet;
    }
}
```